editor/tests/org.fusesource.ide.camel.editor.tests/src/test/java/org/fusesource/ide/camel/editor/utils/MavenUtilsTest.java

```diff
@@ -91,6 +94,7 @@ public void testUpdateMavenDependencyWithNullVersion() throws Exception {
 	}
 
 	@Test
+	@Ignore("Disabled test because its broken due to the model rework FUSETOOLS-2290")
```

Comment:
why ignoring it? the PR is for FUSETOOLS-2944 based on commit message

Reply:
those tests are broken and ignored for a long time now. fixing them is out of scope of this PR

Reply:
raised https://issues.jboss.org/browse/FUSETOOLS-3032 to track the repair task

Reply:
ok, it was disabled at the class level before